cooperative_lanechange/config/parameters.yaml

```diff
@@ -56,3 +56,11 @@ curvature_calc_lookahead_count : 1
 
 # Int: Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
 downsample_ratio : 8
+
+#Int : Acceptable range from the ending downtrack, when planning success can be published, if vehicle has changed lanes
+#Units: meters
+destination_range : 5
```

Comment:
Acceptable range = 5, not sure how why we need value 5 as the range. Can this range = 0? Does this apply to all vehicles?